schema/home/results.js

```diff
@@ -18,8 +18,8 @@ const RESULTS_SIZE = 20;
 
 const moduleResults = {
   active_bids: () => [],
-  iconic_artists: () => {
-    return iconicArtists().then((artists) => {
+  trending_artists: () => {
```

Comment:
I think this is a breaking change because Force uses "iconic_artists" in one place that's hard-coded, in order to render a specific template:

<img width="1237" alt="screen shot 2016-07-14 at 4 22 04 pm" src="https://cloud.githubusercontent.com/assets/1457859/16854485/210f8ae8-49df-11e6-8656-d06dfb83d914.png">